week-4/Homework/mandatory/2-writers.js

```diff
@@ -38,6 +38,11 @@ let writers = [
     alive: true
   }
 ];
+function getName(writers){

```

Comment:
This works really well, good job!

One thing to think about that could make this code a little clearer is the naming. Let's look at this line - both of the names could be a little more clear. Let's start with the parameter - this function is actually only called with _one_ writer, not all of the writers, so `writer` may be a better name than `writers`.

And then the function name - `getName` - can you think about what this function does, and accordingly what may be a better name for it than `getName`?